login-workflow/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v4.0.3 (Aug 9, 2024)
```

Comment:
you can use 4.0.4 or the next release version here with no date (unreleased) 4.0.3 was released in may